projects/ion/src/lib/button/button.component.spec.ts

```diff
@@ -167,6 +167,19 @@ describe('ButtonComponent with dropdown', () => {
       );
     });
 
+    it('should render a single-selection dropdown that close when a option is clicked', async () => {
+      const options = [{ label: 'Option 1' }, { label: 'Option 2' }];
+
+      const button = await sut({
+        label: defaultName,
+        options,
+      });
+
+      fireEvent.click(button);
+
+      expect(screen.queryByTestId('Option 1')).toBeNull();
```

Comment:
```suggestion
      expect(screen.queryByTestId(options[0].label)).toBeNull();
```

This way you not need change the label in 2 places.